stdlib/testing/testdata/yield.flux

```diff
@@ -29,20 +32,23 @@ outData = "
 ,result,table,_start,_stop,_time,_value,_field,_measurement,cpu,host
 ,,0,2018-05-22T19:53:24.421470485Z,2018-05-22T19:54:24.421470485Z,2018-05-22T19:54:16Z,87.88598574821853,usage_idle,cpu,cpu-total,host.local
 "
-
 t_yield = (table=<-) =>
-  table
-    |> sort()
-    |> limit(n: 3)
-    |> yield(name: "1: lowest 3")
-    |> sample(n: 2, pos: 1)
-    |> yield(name: "2: 2nd row")
-
+	(table
+		|> sort()
+		|> limit(n: 3)
+		|> yield(name: "1: lowest 3")
+		|> sample(n: 2, pos: 1)
+		|> yield(name: "2: 2nd row"))
 indata = testing.loadStorage(csv: inData)
-got = indata |> t_yield() |> yield(name: "5")
-want = testing.loadStorage(csv: outData) |> yield(name:"6")
+got = indata
+	|> t_yield()
+	|> yield(name: "5")
+want = testing.loadStorage(csv: outData)
+	|> yield(name: "6")
+
 testing.assertEquals(name: "yield", want: want, got: got)
-testing.test(name: "yield",
-            input: testing.loadStorage(csv: inData),
-            want: testing.loadMem(csv: outData),
-            testFn: t_yield)
+
+test test_yield = () =>
+	({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_yield})
+
```

Comment:
this test is disabled because we don't have a good way to inspect yield() text anymore, since we quit doing byte-for-byte text comparison.  any ideas?